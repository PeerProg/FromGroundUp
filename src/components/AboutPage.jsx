import React from 'react';

const AboutPage = () => {
  return (
    <div className="App">
      <h1>This is the about page</h1>
    </div>
  )
};

export default AboutPage;
