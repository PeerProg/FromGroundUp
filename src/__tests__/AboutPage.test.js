import React from 'react';
import AboutPage from '../components/AboutPage';
import renderer from 'react-test-renderer';

test('About Page Component', () => {
  const component = renderer.create(
    <AboutPage />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
