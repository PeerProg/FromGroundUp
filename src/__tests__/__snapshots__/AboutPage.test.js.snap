// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Component 1`] = `
<div>
  <h1>
    This is the about page
  </h1>
</div>
`;
